src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs

```diff
@@ -1974,6 +1974,43 @@ public async Task TestMetadataOverride()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsReplaceMethodWithProperty)]
+        public async Task IgnoreIfTopLevelNullableIsDifferent()
```

Comment:
What about nested nullability?

Reply:
Ah, like `IEnumerable<string?>`, I'll add tests, idk if the default comparer handles correctly.

Reply:
"Correctly" being it doing magic without me doing anything

Reply:
So it doesn't look like nested types are covered in tests or handled well. The 'Get' and 'Set' never get matched. I filed #37991 to track this, and dumped the two tests that need to be fixed with it.